lib/ui/core/ui/wn_heads_up.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
 import 'package:gap/gap.dart';
-import 'package:supa_carbon_icons/supa_carbon_icons.dart';
+import 'package:whitenoise/ui/core/themes/assets.dart';
 import 'package:whitenoise/ui/core/themes/src/app_theme.dart';
 
 class WnStickyHeadsUp extends StatelessWidget {
```

Comment:
Oopps probably my bad here when adding Wn prefix to existing components, but this file name does not match the widget name. Can be fixed in other issue, just commenting it here to not forget to fix it.